--- a/arch/arm/mach-imx/iomux-v3.c	2016-03-28 15:54:02.664472488 +0300
+++ a/arch/arm/mach-imx/iomux-v3.c	2016-03-28 15:49:09.000000000 +0300
@@ -73,6 +73,22 @@
 }
 EXPORT_SYMBOL(mxc_iomux_v3_setup_multiple_pads);
 
+void mxc_iomux_set_gpr_register(int group, int start_bit, int num_bits,
+				int value)
+{
+	int i = 0;
+	u32 reg;
+	reg = __raw_readl(base + group * 4);
+	while (num_bits) {
+		reg &= ~(1<<(start_bit + i));
+		i++;
+		num_bits--;
+	}
+	reg |= (value << start_bit);
+	__raw_writel(reg, base + group * 4);
+}
+EXPORT_SYMBOL(mxc_iomux_set_gpr_register);
+
 void mxc_iomux_v3_init(void __iomem *iomux_v3_base)
 {
 	base = iomux_v3_base;


--- a/arch/arm/mach-imx/iomux-v3.h	2016-03-28 15:53:50.048357908 +0300
+++ a/arch/arm/mach-imx/iomux-v3.h	2016-03-28 15:48:19.000000000 +0300
@@ -138,5 +138,8 @@
  */
 void mxc_iomux_v3_init(void __iomem *iomux_v3_base);
 
+void mxc_iomux_set_gpr_register(int group, int start_bit, int num_bits,
+				int value);
+
 #endif /* __MACH_IOMUX_V3_H__*/
