--- defconfig	2016-03-31 14:45:27.410174141 +0300
+++ .config_31.03_3	2016-03-31 14:57:23.000000000 +0300
@@ -1819,7 +1819,7 @@
 #
 # Multifunction device drivers
 #
-CONFIG_MFD_CORE=y
+# CONFIG_MFD_CORE is not set
 # CONFIG_MFD_AS3711 is not set
 # CONFIG_PMIC_ADP5520 is not set
 # CONFIG_MFD_AAT2870_CORE is not set
@@ -1829,7 +1829,7 @@
 # CONFIG_MFD_DA9052_SPI is not set
 # CONFIG_MFD_DA9052_I2C is not set
 # CONFIG_MFD_DA9055 is not set
-CONFIG_MFD_MXC_HDMI=y
+# CONFIG_MFD_MXC_HDMI is not set
 # CONFIG_MFD_MC13XXX_SPI is not set
 # CONFIG_MFD_MC13XXX_I2C is not set
 # CONFIG_HTC_EGPIO is not set
@@ -2050,7 +2050,7 @@
 #
 # MXC Vivante GPU support
 #
-# CONFIG_MXC_GPU_VIV is not set
+CONFIG_MXC_GPU_VIV=y
 CONFIG_MXC_IPU_V3=y
 
 #
